plugins/cicd-statistics/api-report.md

```diff
@@ -0,0 +1,182 @@
+//
+// @public (undocumented)
+export const branchTypes: Array<FilterBranchType>;
+
+// Warning: (ae-missing-release-tag) "Build" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
+export interface Build {
+  buildType: FilterBranchType;
+  duration: number;
+  id: string;
+  // (undocumented)
+  raw?: unknown;
+  requestedAt: Date;
+  stages: Array<Stage>;
+  status: FilterStatusType;
+}
+
+// Warning: (ae-missing-release-tag) "BuildWithRaw" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
+export type BuildWithRaw<T = any> = Build & {
+  raw: T;
+};
+
+// Warning: (ae-missing-release-tag) "CicdConfiguration" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
+export interface CicdConfiguration {
+  availableStatuses: ReadonlyArray<FilterStatusType>;
+  defaults: Partial<CicdDefaults>;
+  formatStageName: (parentNames: Array<string>, stageName: string) => string;
+}
+
+// Warning: (ae-missing-release-tag) "CicdDefaults" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
+export interface CicdDefaults {
+  collapsedLimit: number;
+  // (undocumented)
+  filterStatus: Array<FilterStatusType>;
+  // (undocumented)
+  filterType: FilterBranchType<'all'>;
+  lowercaseNames: boolean;
+  normalizeTimeRange: boolean;
+  // (undocumented)
+  timeFrom: Date;
+  // (undocumented)
+  timeTo: Date;
+}
+
+// Warning: (ae-missing-release-tag) "CicdState" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
+export interface CicdState {
+  // (undocumented)
+  builds: Array<Build>;
+}
+
+// Warning: (ae-missing-release-tag) "CicdStatisticsApi" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
+export interface CicdStatisticsApi {
+  // (undocumented)
+  fetchBuilds(options: FetchBuildsOptions): Promise<CicdState>;
+  // (undocumented)
+  getConfiguration(): Promise<Partial<CicdConfiguration>>;
+}
+
+// Warning: (ae-missing-release-tag) "cicdStatisticsApiRef" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export const cicdStatisticsApiRef: ApiRef<CicdStatisticsApi>;
+
+// Warning: (ae-missing-release-tag) "cicdStatisticsPlugin" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export const cicdStatisticsPlugin: BackstagePlugin<
+  {
+    entityContent: RouteRef<undefined>;
+  },
+  {}
+>;
+
+// Warning: (ae-forgotten-export) The symbol "EntityPageCicdCharts" needs to be exported by the entry point index.d.ts
```

Comment:
Would be nice to clean up all the warnings here, but I particularly recommend fixing the missing export warning, as the API Report does not show the complete API if you have these

Reply:
Hidden in an ocean of `ae-missing-release-tag`. Do we need these?

Reply:
Yeah if you wanna go through and add release tags that doesn't hurt. I'd call it optional at this point but it definitely cleans up the API report.